functions/snippets/ci_cd/cloudbuild.yaml

```diff
@@ -0,0 +1,9 @@
+# [START functions_ci_cd_cloud_build]
```

Comment:
Shouldn't there be a license and copyright?

Reply:
Done.